MaxText/convert_gpt3_ckpt_from_paxml.py

```diff
@@ -52,6 +52,7 @@
 import jax
 import gc
 from MaxText import max_logging
+from MaxText import maxtext_checkpointing_utils
```

Comment:
Looks like `from MaxText import max_utils` is no longer needed. Can you remove it from this file and other ones where it isn't needed either?